### PR TITLE
Prototype for sending emojis

### PR DESCRIPTION
**Pull request by rnowak6**
Got a prototype for sending emojis by using a library.
Currently to send an emoji you must, for example, say :smiley: and it produces the unicode emoji : ðŸ˜„ 
Next step would be to figure out how to get it to display actual emojis. Once that is done,
the next step would be to allow users to input emojis rather than typing them out

### CODE REVIEW

File: src/main/java/codeu/controller/ChatServlet.java [hunk 1]
```diff
@@ -163,14 +165,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     
     // this removes any HTML from the message content
     String cleanedMessageContent = processor.process(Jsoup.clean(messageContent, Whitelist.none()));
-
+    
+    
+    
+    String emojis = EmojiParser.parseToUnicode(cleanedMessageContent);
```
Comment (justincuig) [inline src/main/java/codeu/controller/ChatServlet.java]: can user send emojis together with text?

Comment (rnowak6) [reply src/main/java/codeu/controller/ChatServlet.java]: not yet, that would be the next step

**Comment by rnowak6** [comment]
For some reason my recent changes got put into this pull request and I'm not sure how to make them separate

**Comment by rnowak6** [comment]
This is what I was working on this time

conversations
buttons

**Comment by rnowak6** [comment]
As of right now, users are able to go to the all conversations tab and view all public convos. My next step would be making sure that uses can actually open the convos. In the conversation page, I added buttons that set the conversation to private or public. Right now the buttons don't really do much because I was focusing on the overall structure but for the next step I will have them actually working and each private conversation will have a flag so that it is not included in the conversations that other people can join

**Comment by justincuig** [comment]
good progress!

**Pull request merged**